FCCee/Generator/EvtGen/Lb2LbMuMu.dec

```diff
@@ -0,0 +1,21 @@
+Alias   Lb_SIGNAL   Lambda_b0
+Alias   Lbbar_SIGNAL    anti-Lambda_b0
+ChargeConj Lb_SIGNAL  Lbbar_SIGNAL
+
+Alias      MyLambda0      Lambda0
+Alias      Myanti-Lambda0 anti-Lambda0
+ChargeConj MyLambda0      Myanti-Lambda0
+
+#
+Decay Lb_SIGNAL
+  1.000        MyLambda0    mu+    mu-         BTOSLLBALL;
```

Comment:
Got error during generation
```
EvtGen:BTOSLLBALL did not get the correct parent spin
Lb_SIGNAL -> MyLambda0 mu+ mu-  (BTOSLLBALL)
```
Could you double check the implementation of this model?